hdrop/hdrop

```diff
@@ -1,83 +1,106 @@
-  exit 1
-  ;;
-"-h" | "--help")
+print_help() {
   echo ""
-  echo "This script emulates the main feature of tdrop (https://github.com/noctuid/tdrop) in Hyprland, namely:"
+  echo "Usage: hdrop [OPTIONS] [COMMAND]"
   echo ""
-  echo " - if the specified program is not running: launch it and bring it to the foreground."
-  echo " - if the specified program is already running on another workspace: bring it to the current workspace and focus it."
-  echo " - if the specified program is already on the current workspace: move it to workspace 'special:hdrop', thereby hiding it until called up again by hdrop."
+  echo "Arguments:"
+  echo "  [COMMAND]"
+  echo "          The usual command you would run to start the desired program"
   echo ""
-  echo "Several instances of the same program can be run concurrently, if different class names are assigned to each instance. Presently there is support for the following flags:"
+  echo "Options:"
+  echo "  -b, --background"
+  echo "          changes the default behaviour: if the specified program is"
+  echo "          not running, launch it in the background instead of foreground."
+  echo "          Thereafter 'hdrop -b' will work the same as without this flag."
   echo ""
-  echo " -a ('foot' terminal emulator)"
-  echo " --class (all other programs)."
+  echo "  -h, --help"
+  echo "          Print this help message"
   echo ""
-  echo "Please feel free to report missing flags for other programs!"
+  echo "  -V, --version"
+  echo "          Print version"
   echo ""
-  echo "Example bindings in Hyprland config:"
+  echo "Multiple instances:"
   echo ""
-  echo ' bind = $mainMod, b, exec, hdrop librewolf'
-  echo ' bind = $mainMod, x, exec, hdrop kitty --class kitty_1'
-  echo ' bind = $mainMod CTRL, x, exec, hdrop kitty --class kitty_2'
-  echo ' bind = $mainMod, c, exec, hdrop foot -a foot_1'
-  echo ' bind = $mainMod CTRL, c, exec, hdrop foot -a foot_2'
+  echo "Multiple instances of the same program can be run concurrently, if"
+  echo "different class names are assigned to each instance. Presently there is"
+  echo "support for the following flags in the [COMMAND] string:"
   echo ""
-  echo "Note: defining a class name is only necessary when running several instances of the same program."
-  echo ""
-  echo "If you want to run a program on boot and have it wait in the background until called up by hdrop you can use this:"
-  echo ""
-  echo " exec-once = [workspace special:hdrop silent] librewolf"
-  echo ""
-  echo "Dependencies: bash jq hyprland"
-  echo "Part of: https://github.com/hyprwm/contrib"
-  echo "Author: Schweber (https://github.com/Schweber/hdrop)"
-  echo "License: AGPL3"
+  echo " -a         ('foot' terminal emulator)"
+  echo " --class    (all other programs)."
   echo ""
+  echo "See man page for more information"
+}
+
+print_version() {
+  echo "hdrop version: 0.2.1"
```

Comment:
Is `0.2.1` intentional, or was it meant to be `0.2.0`? If it's the former, please also update the version in the PR name, commits and `default.nix`.

Reply:
It was intentional as released 0.2.0 without the manpage etc.
I'll update the other files